index.js

```diff
@@ -23,8 +23,22 @@ function fastifyMysql (fastify, options, next) {
     db.dispose().then(() => done()).catch(done)
   })
 
+  const executeTransaction = async (queries) => {
```

Comment:
This is good, but I'd make some changes:

- Redeclare the `executeTransaction` method using the naive function declaration:
```js
function executeTransaction (queries) {}
```